kettle/README.md

```diff
@@ -100,10 +100,25 @@ gs://<bucket path>: #bucket url
 
 A [postsubmit job](https://github.com/kubernetes/test-infra/blob/master/config/jobs/kubernetes/test-infra/test-infra-trusted.yaml#L203-L210) runs that pushes Kettle on changes.
 
+# PubSub
+
+Kettle `stream.py` leverages Google Cloud [PubSub] to alert on GCS changes. These events are tied to the `gcs-changes` Topic in the `kubernetes-jenkins` project where Prow job artifacts are collated. Each time an artifact is finalized, a PubSub event is triggered and Kettle collects job information when it sees a resource uploaded called `finished.json` (indicating the build completed).
+
+[Subscriptions] are in Kuberenetes Jenkins Build - PubSub.
+- kettle
+- kettle-staging
+
+They are split so that the staging instance does not consume events aimed at production.
+
+### Auth
+For kettle to have permisions, kettle's user needs access. When updating or changing a [Subscription] make sure to add `kettle@k8s-gubernator.iam.gserviceaccount.com` as a `PubSub Editor`.
```

Comment:
nit: gcloud example to copy-paste would be helpful

Reply:
Sure, I used UI but I can try and find the command +1

Reply:
```sh
gcloud pubsub subscriptions add-iam-policy-binding \
  projects/kubernetes-jenkins/subscriptions/kettle-staging \
  --member=serviceAccount:kettle@k8s-gubernator.iam.gserviceaccount.com \
  --role=roles/pubsub.editor
```

Reply:
Added with the command to make the subscriptions.